.github/workflows/yaml-linter.yaml

```diff
@@ -0,0 +1,19 @@
+name: Yaml Lint
```

Comment:
Is there a way to suppress warnings? It would be nice to either fix them or suppress them, similar to errors, or maybe it's not worth it.